---a/net/minecraft/world/biome/BiomeGenStoneBeach.java
+++b/net/minecraft/world/biome/BiomeGenStoneBeach.java
@@ -2,12 +2,12 @@
 
 import net.minecraft.init.Blocks;
 
-public class BiomeGenStoneBeach extends BiomeGenBase
-{
+public class BiomeGenStoneBeach extends BiomeGenBase {
+
     private static final String __OBFID = "CL_00000184";
 
-    public BiomeGenStoneBeach(int p_i45384_1_)
-    {
+    public BiomeGenStoneBeach(int p_i45384_1_) {
         super(p_i45384_1_);
         this.spawnableCreatureList.clear();
         this.topBlock = Blocks.stone.getDefaultState();
@@ -17,4 +17,4 @@
         this.theBiomeDecorator.reedsPerChunk = 0;
         this.theBiomeDecorator.cactiPerChunk = 0;
     }
-}
+}
